### PR TITLE
KAFKA-10225 Increase default zk session timeout for system tests

### PR DESCRIPTION
**Pull request by chia7712**
I'm digging in the flaky system tests and then I noticed there are many flaky caused by following check.
```python
        with node.account.monitor_log(KafkaService.STDOUT_STDERR_CAPTURE) as monitor:
            node.account.ssh(cmd)
            # Kafka 1.0.0 and higher don't have a space between "Kafka" and "Server"
            monitor.wait_until("Kafka\s*Server.*started", timeout_sec=timeout_sec, backoff_sec=.25,
                               err_msg="Kafka server didn't finish startup in %d seconds" % timeout_sec)
```

And the error message in broker log is shown below.
```
kafka.zookeeper.ZooKeeperClientTimeoutException: Timed out waiting for connection while in state: CONNECTING
	at kafka.zookeeper.ZooKeeperClient.waitUntilConnected(ZooKeeperClient.scala:262)
	at kafka.zookeeper.ZooKeeperClient.<init>(ZooKeeperClient.scala:119)
	at kafka.zk.KafkaZkClient$.apply(KafkaZkClient.scala:1880)
	at kafka.server.KafkaServer.createZkClient$1(KafkaServer.scala:430)
	at kafka.server.KafkaServer.initZkClient(KafkaServer.scala:455)
	at kafka.server.KafkaServer.startup(KafkaServer.scala:227)
	at kafka.server.KafkaServerStartable.startup(KafkaServerStartable.scala:44)
	at kafka.Kafka$.main(Kafka.scala:82)
	at kafka.Kafka.main(Kafka.scala)
```

I'm surprised the default timeout of zk connection in system test is only 2 seconds as the default timeout in production is increased to 18s (see https://github.com/apache/kafka/commit/4bde9bb3ccaf5571be76cb96ea051dadaeeaf5c7)
```
config_property.ZOOKEEPER_CONNECTION_TIMEOUT_MS: 2000
```

issue: https://issues.apache.org/jira/browse/KAFKA-10225

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: tests/kafkatest/services/kafka/config.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ class KafkaConfig(dict):
         config_property.PORT: 9092,
         config_property.SOCKET_RECEIVE_BUFFER_BYTES: 65536,
         config_property.LOG_DIRS: "/mnt/kafka/kafka-data-logs-1,/mnt/kafka/kafka-data-logs-2",
-        config_property.ZOOKEEPER_CONNECTION_TIMEOUT_MS: 2000
+        config_property.ZOOKEEPER_CONNECTION_TIMEOUT_MS: 18000
```
Comment (chia7712) [inline tests/kafkatest/services/kafka/config.py]: make it equal to https://github.com/apache/kafka/blob/trunk/config/server.properties#L126

**Comment by abbccdda** [comment]
cc @hachikuji

File: tests/kafkatest/services/kafka/kafka.py [hunk 2]
```diff
@@ -185,7 +185,11 @@ def __init__(self, context, num_nodes, zk, security_protocol=SecurityConfig.PLAI
 
         for node in self.nodes:
             node.version = version
-            node.config = KafkaConfig(**{config_property.BROKER_ID: self.idx(node)})
+            node.config = KafkaConfig(**{
+                config_property.BROKER_ID: self.idx(node),
```
Comment (chia7712) [inline tests/kafkatest/services/kafka/kafka.py:189]: This is another bug that we don't honor the timeout parameters assigned to ```KafkaService```

File: tests/kafkatest/services/kafka/config.py [hunk 3]
```diff
@@ -25,7 +25,8 @@ class KafkaConfig(dict):
         config_property.PORT: 9092,
         config_property.SOCKET_RECEIVE_BUFFER_BYTES: 65536,
         config_property.LOG_DIRS: "/mnt/kafka/kafka-data-logs-1,/mnt/kafka/kafka-data-logs-2",
-        config_property.ZOOKEEPER_CONNECTION_TIMEOUT_MS: 2000
+        config_property.ZOOKEEPER_CONNECTION_TIMEOUT_MS: 5000,
+        config_property.ZOOKEEPER_SESSION_TIMEOUT_MS: 6000
```
Comment (junrao) [inline tests/kafkatest/services/kafka/config.py]: The default ZK session timeout and ZK connection timeout are both 18 secs. So we can just change both to 18secs here.

File: tests/kafkatest/services/kafka/kafka.py [hunk 4]
```diff
@@ -185,7 +185,11 @@ def __init__(self, context, num_nodes, zk, security_protocol=SecurityConfig.PLAI
...
+                config_property.ZOOKEEPER_CONNECTION_TIMEOUT_MS: zk_connect_timeout,
```
Comment (junrao) [inline tests/kafkatest/services/kafka/kafka.py:190]: Could we change the default ZK session timeout and ZK connection timeout in line 101 to 18 secs too?

**Review by junrao — COMMENTED** [review]
@chia7712 : Thanks for the PR. Just a couple of minor comments below.

File: tests/kafkatest/services/kafka/config.py (hunk 3 above)
Comment (chia7712) [reply tests/kafkatest/services/kafka/config.py]: sure!

**Review by junrao — COMMENTED** [review]
@chia7712 : Thanks for the updated PR. LGTM. Will trigger a round of system tests on this PR.

**Comment by junrao** [comment]
System test results. All 9 failures seem unrelated to this PR. http://confluent-kafka-branch-builder-system-test-results.s3-us-west-2.amazonaws.com/2020-07-07--001.1594186925--chia7712--KAFKA-10225--da9265760/report.html

**Review by junrao — APPROVED** [review]

**Pull request merged**